depmapomics/mutations.py

```diff
@@ -71,11 +46,13 @@ def annotateLikelyImmortalized(
     for k, v in Counter(maf[genome_change_col].tolist()).items():
         if v > max_recurrence * leng:
             tocheck.append(k)
-    for val in list(set(tocheck) - set([np.nan])):
-        if (
-            np.nan_to_num(maf[maf[genome_change_col] == val][TCGAlocs], 0).max()
-            < min_tcga_true_cancer
-        ):
+    tocheck = list(set(tocheck) - set([np.nan]))
+    le = len(tocheck)
+    counter = 0
+    for val in tocheck:
+        h.showcount(counter, le)
+        counter += 1
+        if "Y" not in set(maf[maf[genome_change_col] == val][hotspotcol]):
             maf.loc[
```

Comment:
why not just -> mat[~isHotstop & mat.mutname.isin([i for i in Counter(maf.mutname.tolist()).items() if i>max])]

genome_change_col is not enough!